tests/Makefile

```diff
@@ -44,7 +44,17 @@ showparams:
 tools:
 	@$(MAKE) -s -C tools
 
-$(patsubst %.test,%,$(shell ls -d *.test)): basicops tools
+$(patsubst %.test,%,$(shell ls -d *.test)): basicops tools $(SRCHOME)/cdb2sql $(SRCHOME)/cdb2_dump $(SRCHOME)/cdb2_stat
+
+$(SRCHOME)/cdb2sql: $(SRCHOME)/comdb2
+	ln $(SRCHOME)/comdb2 $(SRCHOME)/cdb2sql
+
+$(SRCHOME)/cdb2_dump:  $(SRCHOME)/comdb2
+	ln $(SRCHOME)/comdb2 $(SRCHOME)/cdb2_dump
+
+$(SRCHOME)/cdb2_stat:  $(SRCHOME)/comdb2
+	ln $(SRCHOME)/comdb2 $(SRCHOME)/cdb2_stat
```

Comment:
We add hard links here for tests. However, neither make clean under tests/ nor make clean under $(SRCHOME) cleans up the links.. If after making tests, I do `make clean && make && sudo make install` under $(SRCHOME) and want to make tests again, it will fail to create the links again as they already exist from the last make.

I think we should either add `rm -f` here or clean them up when `make clean`